docs/schema/aws.md

```diff
@@ -2841,3 +2855,281 @@ Representation of an AWS EC2 [Elastic IP address](https://docs.aws.amazon.com/AW
+| ecc\_log\_configuration\_cloud\_watch\_encryption\_enabled | Determines whether to enable encryption on the CloudWatch logs. |
+| ecc\_log\_configuration\_s3\_bucket\_name | The name of the S3 bucket to send logs to. |
+| ecc\_log\_configuration\_s3\_encryption\_enabled | Determines whether to use encryption on the S3 logs. |
+| ecc\_log\_configuration\_s3\_key\_prefix | An optional folder in the S3 bucket to place logs in. |
+| status | The status of the cluster |
+| settings\_container\_insights | If enabled is specified, CloudWatch Container Insights will be enabled for the cluster, otherwise it will be disabled unless the containerInsights account setting is enabled. |
+| capacity\_providers | The capacity providers associated with the cluster. |
+| attachments\_status | The status of the capacity providers associated with the cluster. |
+
+### Relationships
+
+- ECSClusters are a resource under the AWS Account.
+
+        ```
+        (AWSAccount)-[RESOURCE]->(ECSCluster)
+        ```
+
+## ECSContainerInstance
+
+Representation of an AWS ECS [Container Instance](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ContainerInstance.html)
+
+| Field | Description |
+|-------|-------------|
+| firstseen| Timestamp of when a sync job first discovered this node  |
+| lastupdated |  Timestamp of the last time the node was updated |
+| **id** | The ARN of the container instance |
+| region | The region of the container instance. |
+| ec2\_instance\_id | The ID of the container instance. For Amazon EC2 instances, this value is the Amazon EC2 instance ID. For external instances, this value is the AWS Systems Manager managed instance ID. |
+| arn | The ARN of the container instance |
+| capacity\_provider\_name | The capacity provider that's associated with the container instance. |
+| version | The version counter for the container instance. |
+| version\_info\_agent\_version | The version number of the Amazon ECS container agent. |
+| version\_info\_agent\_hash | The Git commit hash for the Amazon ECS container agent build on the amazon-ecs-agent  GitHub repository. |
+| version\_info\_agent\_docker\_version | The Docker version that's running on the container instance. |
+| status | The status of the container instance. |
+| status\_reason | The reason that the container instance reached its current status. |
+| agent\_connected | This parameter returns true if the agent is connected to Amazon ECS. Registered instances with an agent that may be unhealthy or stopped return false. |
+| agent\_update\_status | The status of the most recent agent update. If an update wasn't ever requested, this value is NULL. |
+| registered\_at | The Unix timestamp for the time when the container instance was registered. |
+
+### Relationships
+
+- An ECSCluster has ECSContainerInstances
+
+        ```
+        (ECSCluster)-[HAS_CONTAINER_INSTANCE]->(ECSContainerInstance)
+        ```
```

Comment:
Please mention this relationship under `ECSCluster` too and follow the same pattern for the rest of the resources.